CHANGELOG.rst

```diff
@@ -2,6 +2,42 @@
 CHANGELOG
 =========
 
+2.9.0
+=====
+
+**ENHANCEMENTS**
+
+* Added support for heterogeneous instance types with Slurm Scheduler.
+* Added CLI utility to convert ParallelCluster configuration files from single to Heterogeneous instance types.
+* Extended DCV support to ARM instances.
+* Added support for Torque-Slurm wrapper.
```

Comment:
"Torque-Slurm wrapper"  ?!  -  Add script wrapper to support Torque-like commands with the Slurm scheduler.